### PR TITLE
Update upic from v0.16.3 to v0.16.4

### PR DESCRIPTION
**Pull request by gee1k**
Emergency repair

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/upic.rb [hunk 1]
```diff
@@ -10,9 +10,4 @@
   depends_on macos: '>= :sierra'
 
   app 'uPic.app'
-
-  zap trash: [
```
Comment (vitorgalvao) [inline Casks/upic.rb]: Why remove these?

Comment (gee1k) [reply Casks/upic.rb]: These files do not need to be deleted but are kept

Comment (vitorgalvao) [reply Casks/upic.rb]: > These files do not need to be deleted but are kept

I don’t understand what you mean. Aren’t these valid configuration files? They’ll only be removed if the user `zap`s the cask, not if they `uninstall` or `upgrade`.

Comment (gee1k) [reply Casks/upic.rb]: Oh, it seems I understand it wrong. I will readjust

Sorry for your trouble

**Review by vitorgalvao — APPROVED** [review]

File: Casks/upic.rb (hunk 1 above)
Comment (gee1k) [reply Casks/upic.rb]: I have readjusted, and I am sorry again for the trouble

Comment (vitorgalvao) [reply Casks/upic.rb]: No need to apologise. It was a small and understandable mistake. Thank you for the quick fix.

**Pull request merged**